dev-python/backports-tempfile/backports_tempfile-1.0.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="Backport of new features in Python's tempfile module"
+DESCRIPTION="This package provides backports of new features in Python's \
+tempfile_ module under the backports_ namespace."
+HOMEPAGE="https://github.com/pjdelport/backports.tempfile/"
+COPYRIGHT="2016-2017 Pi Delport"
+LICENSE="PSF-2.3"
+REVISION="1"
+SOURCE_URI="https://github.com/PiDelport/backports.tempfile/archive/refs/tags/v$portVersion.tar.gz"
+CHECKSUM_SHA256="597de6793dd1abcf2aea80ae81e8cbf0af4afb117cd48fa1dc3862f85f48c389"
+SOURCE_FILENAME="backports-tempfile-v$portVersion.tar.gz"
+SOURCE_DIR="backports.tempfile-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	$portName = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	"
+
+PYTHON_PACKAGES=(python3 python38 python39)
+PYTHON_VERSIONS=(3.7 3.8 3.9)
+for i in "${!PYTHON_PACKAGES[@]}"; do
+pythonPackage=${PYTHON_PACKAGES[i]}
+pythonVersion=${PYTHON_VERSIONS[$i]}
+eval "PROVIDES_${pythonPackage}=\"\
+	${portName}_$pythonPackage = $portVersion\
+	\"; \
+REQUIRES_$pythonPackage=\"\
+	haiku\n\
+	cmd:python$pythonVersion\
+	\""
+BUILD_REQUIRES="$BUILD_REQUIRES
+	setuptools_scm_$pythonPackage"
+BUILD_PREREQUIRES="$BUILD_PREREQUIRES
+	cmd:git
```

Comment:
cmd:git shouldn't be required

Reply:
Build fails without it (seems a requirement for the source

Reply:
did you try with SETUPTOOLS_SCM_PRETEND_VERSION?
https://github.com/haikuports/haikuports/blob/master/dev-python/iniconfig/iniconfig-1.1.1.recipe#L54

Reply:
Not yet, will need to give it a run :) thanks for the pointer!

Reply:
Works, thanks @korli !